es-design-system/README.md

```diff
@@ -3,20 +3,3 @@
 ## Documentation
 
 To get started working with the design system locally see the [root README.md](../README.md)
-
-### The es-design-system Documentation
```

Comment:
Moved to https://energysage.atlassian.net/wiki/spaces/DSE/pages/208568321/Deploy+Design+System+Documentation